src/internal/m365/collection/drive/collections_tree_test.go

```diff
@@ -471,40 +442,818 @@ func (suite *CollectionsTreeUnitSuite) TestCollections_GetTree() {
 				errs           = fault.New(true)
 			)
 
-			colls, canUsePrevBackup, err := c.getTree(
+			_, _, err := c.getTree(
 				ctx,
 				prevMetadata,
 				globalExcludes,
 				errs)
 
 			test.expect.err(t, err, clues.ToCore(err))
 			// TODO(keepers): awaiting implementation
+			// assert.Empty(t, colls)
+			// assert.Equal(t, test.expect.skips, len(errs.Skipped()))
+			// test.expect.canUsePrevBackup(t, canUsePrevBackup)
+			// test.expect.counts.Compare(t, c.counter)
+
+			// if err != nil {
+			// 	return
+			// }
+
+			// for _, coll := range colls {
+			// 	collPath := fullOrPrevPath(t, coll)
+
+			// 	if collPath.String() == metadataPath.String() {
+			// 		compareMetadata(
+			// 			t,
+			// 			coll,
+			// 			test.expect.deltas,
+			// 			test.expect.prevPaths)
+
+			// 		continue
+			// 	}
+
+			// 	test.expect.collAssertions.compare(t, coll, globalExcludes)
+			// }
+
+			// test.expect.collAssertions.requireNoUnseenCollections(t)
+		})
+	}
+}
+
+// This test is primarily aimed exercising the full breadth of single-drive delta enumeration
+// and broad contracts.
+// mMre granular testing can be found in the lower level test functions below.
```

Comment:
nit: minor spelling fix
```suggestion
// More granular testing can be found in the lower level test functions below.
```

Reply:
bBrgle